src/components/markdown-textarea.js

```diff
@@ -1,34 +1,37 @@
 import React from 'react';
 import { Input } from 'antd';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import Markdown from './markdown.js';
 
 const { TextArea } = Input;
 
-function MarkdownTextarea({ value, onChange }) {
+function MarkdownTextarea({ value, disabled, onChange }) {
 
   const handleOnChange = newValue => {
     onChange(newValue);
   };
 
   return (
     <div className="MarkdownTextarea">
-      <TextArea style={{ height: '100%', resize: 'none' }} value={value} onChange={handleOnChange} />
-      <div className="MarkdownTextarea-preview">
+      <TextArea style={{ height: '100%', resize: 'none' }} value={value} disabled={disabled} onChange={handleOnChange} />
+      <div className={classNames('MarkdownTextarea-preview', { 'is-disabled': disabled })}>
         <Markdown renderMedia>{value}</Markdown>
       </div>
     </div>
   );
 }
 
-MarkdownTextarea.defaultProps = {
-  onChange: () => '',
-  value: ''
-};
-
 MarkdownTextarea.propTypes = {
+  disabled: PropTypes.bool,
   onChange: PropTypes.func,
   value: PropTypes.string
 };
 
+MarkdownTextarea.defaultProps = {
+  disabled: false,
+  onChange: () => '',
```

Comment:
As we do not do anything with the return value `() => {}` would be enough (also when the prop is set it won't probably return a string)

Reply:
Some old code I moved around, I agree and changed it.